linter/src/mock_api/mock_api.cpp

```diff
@@ -2,25 +2,11 @@
 
 namespace nutc {
 namespace mock_api {
-std::function<bool(const std::string&, const std::string&, float, float)>
 
+std::function<bool(const std::string&, const std::string&, float, float)>
 getMarketFunc()
 {
-    return [](const std::string& side,
-              const std::string& ticker,
-              float quantity,
-              float price) {
-        log_i(
-            mock_api,
-            "Mock API: Placing order side {} ticker {} quantity {} price "
-            "{}",
-            side,
-            ticker,
-            quantity,
-            price
-        );
-        return true;
-    };
+    return [](const std::string&, const std::string&, float, float) { return true; };
```

Comment:
why does this function return a closure that doesn't capture anything???

delete these two files
and add this to `lint.cpp`:

```cpp
namespace {

bool 
mock_market_func(const std::string&, const std::string&, float, float)
{
    return true;
}

} // namespace

Reply:
then you can call `create_api_module(mock_market_func)`.